### ISSUE TITLE
Rightclick menu appears for short period of time and option gets clicked automatically

### ISSUE THREAD

**Issue by kroeliebuschie** [akhodakivskiy/VimFx#842]
- [x] I've searched for duplicates – also closed issues
- [x] I've looked at the [change log](https://github.com/akhodakivskiy/VimFx/blob/master/CHANGELOG.md)
- [x] I've provided the below details:

VimFx version: 0.20.4
Firefox version: 50.0
OS: Ubuntu 16.04 LTS
Keyboard language: US
Example URL (if any): -

This is … (pick one!)

- [x] A bug report
  - [x] I've filled in steps to reproduce below
- [ ] A feature request
  - [ ] I've given a **detailed** _example_ below
- [ ] Somehting else

This bug is about … (pick one!)

- [ ] A missing hint marker on a specific page
- [ ] VimFx not recognizing a text input on a specific page
- [x] Something else

Steps to reproduce:

1. go to wikipedia for example https://en.wikipedia.org/wiki/Vim_(text_editor)
2. rightclick the vim logo

Expected result: 
rightclicking an image in wikipedia should result in a menu rightclickmenu 

Actual result: 
the rightclick menu appears for a short period of time and it seems "open in new window" gets  clicked. This behavior does not occur when vimfx is disabled

**Comment by kroeliebuschie** [comment]
only happens with the logo in infoboxes in wikipedia, like the blue logo in the infobox here:https://en.wikipedia.org/wiki/Telegram_(software)

**Comment by lydell** [comment]
- I _can_ reproduce the problem with the same setup as you, but only _sometimes._
- But to me it seems to happen even with VimFx disabled.
- I have not been able to reproduce the problem in Firefox Developer Edition (52).
- I found this, which may or may not be related:

  1. Press and hold the right mouse button. The context menu opens.
  2. Move the mouse pointer on top of a menu item.
  3. Release the right mouse button. The hovered menu item is clicked.

Closing this until more information is found. Right now, it doesn't even look like a VimFx bug.
